### DISCUSSION TITLE
2008/05/29/%E8%B0%B7%E6%AD%8C%E9%87%91%E5%B1%B1%E8%AF%8D%E9%9C%B8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#221] [Announcements]
# 2008/05/29/%E8%B0%B7%E6%AD%8C%E9%87%91%E5%B1%B1%E8%AF%8D%E9%9C%B8

最近一年一直用的是有道词典，周围的人也渐渐放弃了金山词霸。本来以为金山词霸开始没落了，谁知它最近跟 Google 合作，搞出来一个谷歌金山词霸，大有中兴之势。

https://blog.qizhen.xyz/2008/05/29/%E8%B0%B7%E6%AD%8C%E9%87%91%E5%B1%B1%E8%AF%8D%E9%9C%B8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Li
2008年06月24日 15:21 编辑
那为什么不直接用有道的呢？

回复
NeoYu
2008年06月2日 09:08 编辑
不太赞同你的观点，
的确，谷歌金山词霸是有些地方做的比有道好，但是不全部，比如，Google的很多中文名字查询就不咋地，还有屏幕取词，都不咋地，
起码不如有道做的好。
 

回复
HA
2008年05月30日 20:36 编辑
网上直接DOWN程序？

回复
